modules/localgov_workflows_notifications/localgov_workflows_notifications.module

```diff
@@ -0,0 +1,152 @@
+<?php
+
+/**
+ * @file
+ * Primary module hooks for LocalGov Review Notifications module.
+ */
+
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Field\FieldStorageDefinitionInterface;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_entity_base_field_info().
+ */
+function localgov_workflows_notifications_entity_base_field_info(EntityTypeInterface $entity_type) {
+  $fields = [];
+
+  // Add service contacts field to nodes.
+  if ($entity_type->id() === 'node') {
+    $fields['localgov_service_contacts'] = BaseFieldDefinition::create('entity_reference')
+      ->setLabel('Service contacts')
+      ->setSetting('target_type', 'localgov_service_contact')
+      ->setSetting('handler', 'service_contact_reference')
+      ->setCardinality(FieldStorageDefinitionInterface::CARDINALITY_UNLIMITED)
+      ->setDisplayOptions('form', [
+        'type' => 'entity_reference_autocomplete',
+        'settings' => [
+          'match_operator' => 'CONTAINS',
+          'size' => 60,
+          'autocomplete_type' => 'tags',
+          'placeholder' => '',
+        ],
+      ])
+      ->setDisplayConfigurable('form', TRUE);
+  }
+
+  return $fields;
+}
+
+/**
+ * Implements hook_form_alter().
+ */
+function localgov_workflows_notifications_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+
+  // Add service contacts field advanced group on node edit forms.
+  if (isset($form['localgov_service_contacts']) && isset($form['advanced'])) {
+    $form['service_contacts'] = [
+      '#type' => 'details',
+      '#title' => t('Service contacts'),
+      '#description' => t('Search for a service contact by name or email.'),
+      '#group' => 'advanced',
+      '#open' => TRUE,
+      '#weight' => -4,
```

Comment:
How is this going to play if people have altered the form already? Thinking the weight mainly, could it end up hard coded somewhere unintended? Or as it's in advanced is it just maybe irritating, but not important?

Reply:
Not sure now many people form alter the advanced setting, but the weight is there to move is below the review date widget which has a weight of -5 https://github.com/localgovdrupal/localgov_workflows/blob/2f1424ef47e64b7ff3ee04c3a25577d204b4200d/modules/localgov_review_date/localgov_review_date.module#L26